web-components/src/components/favorite/scss/favorite.scss

```diff
@@ -0,0 +1,73 @@
+:host {
+  @include flex($display: inline-flex);
+}
+
+.md-favorite {
+  @include flex($display: inline-flex);
+  border-radius: $favorite-border;
+  height: $favorite-size;
+  padding: calc(#{$favorite-intent} * 1.5);
+  padding-top: calc(#{$favorite-intent} * 2.5);
```

Comment:
nit: spell check, did you mean "indent" ?

Reply:
Yes, Thanks! Changed.